doc/rst/usingchapel/prereqs.rst

```diff
@@ -152,19 +167,31 @@ We have used the following commands to install the above prerequisites:
       sudo dnf install llvm-devel clang clang-devel
 
 
-  * FreeBSD 12.2, 12.3, 13.1::
+  * Fedora 37, 38::
+
+      # Note: CHPL_LLVM=system is not working yet on Fedora 37, 38
+      sudo dnf install gcc gcc-c++ m4 perl python3 python3-devel bash make gawk git cmake
+      sudo dnf install which diffutils
+
+
+  * FreeBSD 12.2, 12.4, 13.1::
 
       sudo pkg install gcc m4 perl5 python3 bash gmake gawk git pkgconf cmake
       sudo pkg install llvm13
 
 
+  * Homebrew::
+
+      brew install cmake python gmp llvm@14
```

Comment:
The homebrew line seems a bit weird considering everything else on this list is an operating system, right?  Maybe it should list Mac OS versions instead?

Reply:
Right, but in terms of "how do I install dependencies", it is answering "How do I install the dependencies on Homebrew". Note that Homebrew can run on linux, so it's not necessarily the case that somebody interested in Homebrew is using a Mac (or that they will look in a Mac section).

If we listed Mac OS X versions, I would also expect the instructions to describe how to install Homebrew, which is not something I wanted to get in to. But, anyway, we already have documentation about Mac OS X that suggests `brew install chapel` which makes these prerequisites irrelevant, in that use case.

I'm half a mind to just remove this section so if it is a source of contention I'll just remove it. We didn't seem to need it in the last release.